package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "specification",
+  "version": "1.0.0",
```

Comment:
can you make this version 0.0.1?  Just in case we do end up versioning through package.json, it would be nice to have some flexibility with the semver.

Reply:
Fixed in https://github.com/fiatconnect/specification/pull/41/commits/8da2611c5af1537847bcc5f79b69e87c39f82697!